powerquery-docs/connectors/google-looker.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Looker connector
+description: Provides basic information and prerequisites for the Looker connector for Power Query.
+author: kenrickyap
+ms.topic: conceptual
+ms.date: 1/02/2024
+ms.author: dougklo
+---
+
+# Looker
+
+![Here's the Looker logo](./media/index/looker-64.png)
```

Comment:
You need to add the note that indicates that this connector is being supplied by Google and where to get support. The note should read something like this (in Markdown):

\> \[\!NOTE\]
\> The following connector article is provided by Google, the owner of this connector and a member of the Microsoft Power Query Connector Certification Program. If you have questions regarding the content of this article or have changes you would like to see made to this article, visit the Google website and use the support channels there.